.reek

```diff
@@ -22,17 +26,26 @@ TooManyMethods:
 UncommunicativeMethodName:
   exclude:
     - Users::PhoneConfirmationController
+    - render_401
 UnusedPrivateMethod:
   exclude:
     - ApplicationController
     - ActiveJob::Logging::LogSubscriber
+    - SamlIdpController
     - Users::PhoneConfirmationController
     - UpdateUserProfileForm
 UtilityFunction:
   public_methods_only: true
   exclude:
     - AnalyticsEventJob#perform
     - SmsSenderConfirmationJob
+
+'spec':
```

Comment:
This means apply the following rules to the `spec` path. Yes, the last line should be removed, but I wasn't sure if it would result in Code Climate reporting issues in the spec files, which would be unrelated to this PR. I should probably remove this change from this PR and do a separate one that configures Reek for the specs and fixes the spec offenses.